apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -54,20 +57,35 @@ export const fetchWorkflows = async (params: {
 
 export type TWorkflowById = z.output<typeof WorkflowByIdSchema>;
 
+export const BaseWorkflowDefinition = ObjectWithIdSchema.extend({
+  name: z.string(),
+  version: z.number(),
+  variant: z.string().default(WorkflowDefinitionVariant.DEFAULT),
+  contextSchema: z.record(z.any(), z.any()).nullable(),
+  documentsSchema: z.array(z.any()).optional().nullable(),
+  config: z.record(z.any(), z.any()).nullable(),
+  definition: z.record(z.any()),
+  transitionSchema: z
+    .array(
+      z.object({
+        state: z.string(),
+        schema: z.record(z.any()),
+        additionalParameters: z.record(z.any()).optional(),
+      }),
+    )
+    .nullable(),
```

Comment:
Should be passed by more team members but this seems to be a good point to group schemas under a single property..